pysbagen/generators/file.py

```
@@ -1,35 +1,91 @@
+            str(path),
+            "-f",
+            "f32le",
+            "-acodec",
+            "pcm_f32le",
+            "-ac",
+            "2",
+            "-ar",
+            str(self.sample_rate),
+            "pipe:1",
+        ]
+        result = subprocess.run(command, capture_output=True, check=False)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)